sycl/test-e2e/bindless_images/3_channel_format.cpp

```diff
@@ -0,0 +1,109 @@
+// REQUIRES: cuda
+
+// RUN: %{build} -o %t.out
+// RUN: %t.out
+
+#include <iostream>
+#include <sycl/detail/core.hpp>
+
+#include <sycl/ext/oneapi/bindless_images.hpp>
+
+// Uncomment to print additional test information
+// #define VERBOSE_PRINT
+
+class image_kernel;
+
+namespace syclexp = sycl::ext::oneapi::experimental;
+
+int main() {
+  sycl::device dev;
+  sycl::queue q(dev);
+  auto ctxt = q.get_context();
+
+  constexpr size_t width = 512;
+  std::vector<float> out(width);
+  std::vector<float> expected(width);
+  std::vector<sycl::float3> dataIn(width);
+  float exp = 512;
+  for (int i = 0; i < width; i++) {
+    expected[i] = exp;
+    dataIn[i] = sycl::float3(exp, width, i);
+  }
+
+  try {
+    // Main point of this test is to check creating an image
+    // with a 3-channel format
+    syclexp::image_descriptor desc({width}, 3, sycl::image_channel_type::fp32);
+
+    syclexp::image_mem imgMem(desc, dev, ctxt);
+
+    q.ext_oneapi_copy(dataIn.data(), imgMem.get_handle(), desc);
+    q.wait_and_throw();
+
+    // Some backends don't support 3-channel formats
+    // We still try to create the image,
+    // but we expect it to fail with UR_RESULT_ERROR_UNSUPPORTED_IMAGE_FORMAT
+    syclexp::unsampled_image_handle imgHandle =
+        sycl::ext::oneapi::experimental::create_image(imgMem, desc, dev, ctxt);
+
+    sycl::buffer<float> buf(out.data(), width);
+
+    q.submit([&](sycl::handler &cgh) {
+      sycl::accessor outAcc{buf};
```

Comment:
```suggestion
      sycl::accessor outAcc{buf, cgh};
```
otherwise there is error on dg2: `Unexpected SYCL exception: placeholder accessor must be bound by calling handler::require() before it can be used.`